Rpackage/R/mmif.cv.predict.R

```diff
@@ -0,0 +1,40 @@
+#' The Predict Function for Cross Validation of Random Forest
+#'
+#' Fits the new data into the mmif.cv model to give prediction values
+#'
+#' @param object Object obtained from \code{"mmif.cv()"}
+#' @param newdata an optional data frame containing the testing data which is to be predicted.
+#' @param perm an optional character vector of variable names. 
+#' 
+#' @return Predictions Average output of each tree in the forest
+#' 
+#' @author Toby Dylan Hocking, Alexandre Drouin, Torsten Hothorn, Parismita Das
+#' 
+#' @examples
+#' library(mmit)
+#' target.mat <- rbind(
+#'   c(0,1), c(0,1), c(0,1),
+#'   c(2,3), c(2,3), c(2,3))
+#' 
+#' feature.mat <- rbind(
+#'   c(1,0,0), c(1,1,0), c(1,2,0),
+#'   c(1,3,0), c(1,4,0), c(1,5,0))
+#' 
+#' colnames(feature.mat) <- c("a", "b", "c")
+#' feature.mat <- data.frame(feature.mat)
+#' 
+#' fit <- mmif.cv(feature.mat, target.mat)
+#' 
+#' pred <- mmif.cv.predict(fit)
+#' 
+mmif.cv.predict <- function(object, newdata = NULL, perm = NULL){
+  
+  all_pred <- NULL
+  forest <- object$best_estimator
```

Comment:
hi @parismita  in order to get `predict(mmif.obj, newX)` working, you need to change the name of these functions. the S3 class system requires that the functions be named as method_name.class_name, so in this case it is predict.mmif and predict.mmif.cv


after you have defined predict.mmif, you can you it in the definition of predict.mmif.cv, e.g.
```r
predict.mmif.cv <- function(object, X, ...){
  predict(object$best_estimator, X, ...)
}
```

Reply:
ok